Watchman.Discord/Areas/Responses/Services/ResponsesMessageService.cs

```diff
@@ -0,0 +1,79 @@
+﻿using Devscord.DiscordFramework.Commons.Extensions;
+using Devscord.DiscordFramework.Framework.Commands.Responses;
+using Devscord.DiscordFramework.Services;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.InteropServices;
+using System.Text;
+using System.Threading.Tasks;
+using Devscord.DiscordFramework.Middlewares.Contexts;
+
+namespace Watchman.Discord.Areas.Responses.Services
+{
+    public class ResponsesMessageService
+    {
+        private const string DESCRIPTION = "dokumentacja:\nhttps://watchman.readthedocs.io/pl/latest/135-services-in-framework/";
+        private readonly ResponsesGetterService _responsesDatabase;
+        private readonly EmbedMessageSplittingService _embedMessageSplittingService;
+
+        public ResponsesMessageService(ResponsesGetterService responsesDatabase, EmbedMessageSplittingService embedMessageSplittingService)
+        {
+            this._responsesDatabase = responsesDatabase;
+            this._embedMessageSplittingService = embedMessageSplittingService;
+        }
+
+        public async Task PrintResponses(string commandArgument, Contexts contexts)
+        {
+            if (commandArgument == "default")
```

Comment:
myśle że dałoby sie to zrobić switchem 

ten z c# 8 wygląda ładnie

Reply:
```suggestion
            Action tmp = commandArgument switch
            {
                "default" => async () => await _embedMessageSplittingService.SendEmbedSplitMessage("Domyślne responses:", DESCRIPTION, GetDefaultResponses(), contexts),
                "custom" => async () => await _embedMessageSplittingService.SendEmbedSplitMessage("Nadpisane responses:", DESCRIPTION, GetCustomResponses(contexts.Server.Id), contexts),
                _ => async () => await _embedMessageSplittingService.SendEmbedSplitMessage("Wszystkie responses:", DESCRIPTION, GetAllResponses(), contexts),
            };
```

Reply:
To ma tak wyglądać czy użyć zwykłych switchy?

Reply:
tak jest ok